src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -75,7 +75,7 @@
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11937" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.2" />
```

Comment:
Where does this version exist? I only see [5.0.0-beta.2-11939](https://www.myget.org/feed/azure-appservice/package/nuget/Microsoft.Azure.WebJobs.Host.Storage/5.0.0-beta.2-11939) on the feed? Shouldn't you be referencing that version?

Reply:
Changed to `5.0.0-beta.2-11939` - was going to hold off this PR till we got the official nuget release for 5.0.0-beta.2 but as discussed offline, we should be good to just use the prerelease!